### DISCUSSION TITLE
Random Survival Forest

### DISCUSSION THREAD

**Discussion by bessierue** [kogalur/randomForestSRC#435] [Q&A]
Good day, 
I just want to ask, i ran a random forest survival model with 129 instances and 16 features. I did hyperparameter tuning to get best parameters. Is it still necessary to perform cross validation? If so how do I do this?

**Comment by wymin576** [comment]
I think you need not perform cross validation. the performance indicator using the OOB samples is sufficient.

**Reply by bessierue** [reply]
Thank you very much for the feedback. I appreciate. Would tuning of the nodesize only be sufficcient though for the same 129 instances with 16 features?
